debian/ubuntu-advantage-tools.postinst

```diff
@@ -37,6 +37,8 @@ OLD_CLIENT_FIPS_PPA="private-ppa.launchpad.net/ubuntu-advantage/fips/ubuntu"
 UA_TIMER_NAME="ua-timer.timer"
 OLD_MESSAGING_TIMER="ua-messaging.timer"
 OLD_MESSAGING_TIMER_MASKED_LOCATION="/etc/systemd/system/timers.target.wants/$OLD_MESSAGING_TIMER"
+OLD_LICENSE_CHECK_PATH="ua-license-check.path"
+OLD_LICENSE_CHECK_PATH_MASKED_LOCATION="/etc/systemd/system/multi-user.target.wants/$OLD_LICENSE_CHECK_PATH"
```

Comment:
Since we will need to perform this for the timer as well, maybe we should create a/ constant for this the path `etc/systemd/system/multi-user.target.wants/`

Reply:
I noticed that this path is different depending on what target triggers it (multi-user.target.wants vs timers.target.wants) so I think if we notice that it is repeating in a future PR then we should extract it into a variable at that time.